src/main/java/dk/kb/present/api/v1/impl/DsPresentApiServiceImpl.java

```diff
@@ -149,4 +149,28 @@ public StreamingOutput getRecords(String origin, Long mTime, Long maxRecords, St
         }
     }
 
+    @Override
+    public StreamingOutput getRawRecords(String origin, Long mTime, Long maxRecords, Boolean asJsonLines) {
```

Comment:
A method named `getRawRecords` that calles another method named `getRecordsRaw` :-)